apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryDbType.cs

```diff
@@ -36,6 +36,18 @@ public enum BigQueryDbType
         /// </summary>
         Timestamp,
         /// <summary>
+        /// A civil date in the Gregorian calendar.
+        /// </summary>
+        Date,
+        /// <summary>
+        /// A civil time-of-day with up to microsecond precision.
```

Comment:
"Up to microsecond precision" is a bit ambiguous, that could mean that it only supports hour precision. What's the minimum guaranteed precision?